cranelift/codegen/src/isa/mod.rs

```diff
@@ -48,16 +48,17 @@ pub use crate::isa::call_conv::CallConv;
 pub use crate::isa::constraints::{
     BranchRange, ConstraintKind, OperandConstraint, RecipeConstraints,
 };
+pub use crate::isa::enc_tables::Encodings;
 pub use crate::isa::encoding::{base_size, EncInfo, Encoding};
 pub use crate::isa::registers::{regs_overlap, RegClass, RegClassIndex, RegInfo, RegUnit};
 pub use crate::isa::stack::{StackBase, StackBaseMask, StackRef};
 
 use crate::binemit;
 use crate::flowgraph;
 use crate::ir;
-use crate::isa::enc_tables::Encodings;
-#[cfg(feature = "unwind")]
 use crate::isa::fde::RegisterMappingError;
+#[cfg(feature = "unwind")]
```

Comment:
Unfortunately I just caught this.  I  believe this accidentally moved the `#[cfg(feature = "unwind")]`.

I have a PR that this merged conflicted with, so I'll correct it there.

Reply:
Yikes, that was likely a rebase failure on my part. Sorry about that. (I suppose I got lucky as `unwind` is enabled by default.) Thanks for fixing!